src/features/projects/components/maps/Credits.tsx

```diff
@@ -39,95 +46,157 @@ export default function Credits({ setCurrencyCode }: Props): ReactElement {
                     {`🌐 ${language ? language.toUpperCase() : ''
                         } • ${selectedCurrency}`}
                 </div>
-                {(process.env.TENANT === "ttc" || process.env.TENANT === "planet") ? < a
+                }
+                {(process.env.TENANT === "ttc" || process.env.TENANT === "planet") && (!isEmbed)? < a
                     rel="noopener noreferrer"
                     href={`https://www.thegoodshop.org/de/shop/`}
                     target={'_blank'}
                 >
                     {t('common:shop')}
                 </a> : null}
+               
                 {/* {config.statusURL ? <a
-                    rel="noopener noreferrer"
-                    href={config.statusURL}
-                    target={'_blank'}
-                >
-                    {t('common:status')}
-                </a> : null} */}
-                <a
-                    rel="noopener noreferrer"
-                    href={`https://status.pp.eco/`}
-                    target={'_blank'}
-                >
-                    {t('common:status')}
-                </a>
-                <a
-                    rel="noopener noreferrer"
-                    href={`https://pp.eco/legal/${i18n.language}/imprint`}
-                    target={'_blank'}
-                >
-                    {t('common:imprint')}
-                </a>
-                <a
-                    rel="noopener noreferrer"
-                    href={`https://pp.eco/legal/${i18n.language}/privacy`}
-                    target={'_blank'}
-                >
-                    {t('common:privacy')}
-                </a>
-                <a
-                    rel="noopener noreferrer"
-                    href={`https://pp.eco/legal/${i18n.language}/terms`}
-                    target={'_blank'}
-                >
-                    {t('common:terms')}
-                </a>
+=======
```

Comment:
You left some code from a merge conflict here and commented it, why didn't you remove it?